index.html

```diff
@@ -1930,6 +1925,43 @@ <h3>
           </li>
         </ol>
       </section>
+      <section>
+        <h3>
+          <code>splashscreens</code> member
+        </h3>
+        <p>
+          A <dfn>splash screen</dfn> is a graphical representation (usually an
+          image on a background color) that is presented as the end user before
+          the user agent is able render content. A splash screen is intended to
```

Comment:
a few typos caught:
- `presented as the end user ` -> `presented to the end user`
- `is able render` → `is able to render`